vowpalwabbit/cb_adf.cc

```diff
@@ -188,6 +188,8 @@ void output_example(vw& all, cb_adf& c, example& ec, v_array<example*>* ec_seq)
   float loss = 0.;
 
   uint32_t action = ec.pred.a_s[0].action;
+  ec.pred.a_s.delete_v();
```

Comment:
Yea, I figured I was doing it wrong, it didn't feel right when I was writing it.  Can you give me some pointers about how I could do this better?